tests/xtro-sharpie/common.pending

```diff
@@ -77,6 +77,15 @@
 !missing-selector! SKVideoNode::initWithFileNamed: not bound
 !missing-selector! SKVideoNode::initWithURL: not bound
 
+## Do we really need these static factory methods, instead of the nice construcotrs we have in C#?
+## Only reason Apple exposed these is because [[foo alloc] init] is annoying
```

Comment:
Agree with @spouliot comments, also I have a small plus of binding them. 

 > Only reason Apple exposed these is because [[foo alloc] init] is annoying

This is not always true 😉 one because they can do `[foo new]` but the really bit I want to add is that sometimes the factory methods have a slight difference than the `init*`. From my old component bindings days there where some libraries that used this factory methods to drive a slightly different behaviour that was only obvious until you read the documentation. I remember a parser library that I bound that the factory method was case sensitive on data and the init was case insensitive.